src/prefect/cli/kubernetes.py

```diff
@@ -1,156 +0,0 @@
-"""
-Command line interface for working with Prefect on Kubernetes
-"""
```

Comment:
Removed this file because the recommended way to run the server/worker is via our published Helm chart.